apps/compound/src/utils/networks.ts

```diff
@@ -0,0 +1,29 @@
+enum CHAINS {
+  MAINNET = 1,
+  MORDEN = 2,
+  ROPSTEN = 3,
+  RINKEBY = 4,
+  GOERLI = 5,
+  KOVAN = 42,
+  XDAI = 100,
+  ENERGY_WEB_CHAIN = 246,
+  VOLTA = 73799,
+  POLYGON = 137,
```

Comment:
Here you defined this enum, but you didn't do it on any other app, just changing the `CHAINS`for a plain `number`

When the app has network dependant configuration I would still use this enum instead of number, can make more explicit which networks are supported on each app